backend/app/integrations/mcp/client.py

```
@@ -0,0 +1,51 @@
+
+        payload = {"method": method, "params": params}
+        headers = {"Content-Type": "application/json"}
+        if self.api_key:
+            headers["Authorization"] = f"Bearer {self.api_key}"
+
+        async with aiohttp.ClientSession() as session:
+            try:
+                async with session.post(self.server_url, headers=headers, json=payload, timeout=timeout) as resp:
+                    text = await resp.text()
+                    if resp.status >= 400:
+                        raise MCPClientError(f"MCP HTTP {resp.status}: {text}")
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)